Open Judge System/Web/OJS.Web/Areas/Administration/Controllers/ParticipantsController.cs

```diff
@@ -205,5 +206,40 @@ public JsonResult UpdateParticipantAnswer([DataSourceRequest]DataSourceRequest r
 
             return this.GridOperation(request, model);
         }
+
+        public ActionResult NormalizeParticipants()
+        {
+            var problems = this.Data.Problems.All().ToList();
+            
+            foreach (var problem in problems)
+            {
+                var data = new OjsData();
```

Comment:
Creating a new instance of OjsData (and consequently of the underlying DbContext) speeds up the execution of the queries tremendously (upward of **x2200** times faster) and ensures no System.OutOfMemory exception is thrown.
Same OjsData instance -> 10 affected rows for 20 minutes
New OjsData for every iteration -> 11k affected rows in about 5 minutes